domaintools/api.py

```diff
@@ -436,8 +436,16 @@ def iris_detect_watched_domains(self, monitor_id=None, escalation_types=None, tl
                              limit=limit, items_path=('watchlist_domains',), response_path=(), **kwargs)
 
     def iris_detect_manage_watchlist_domains(self, watchlist_domain_ids, state, **kwargs):
-        """Changes the watch state of a list of domains by their Iris Detect monitor ID.
+        """Changes the watch state of a list of domains by their Iris Detect domain ID.
```

Comment:
There are two ids that usually get shown. We want to use the domain ID and not the monitor ID.